vision_agent/tools/tools.py

```diff
@@ -455,7 +458,7 @@ def florence2_sam2_image(
 def florence2_sam2_video_tracking(
     prompt: str,
     frames: List[np.ndarray],
-    chunk_length: Optional[int] = 3,
+    chunk_length: Optional[int] = 10,
```

Comment:
@dillonalaird why do you want to increase this value? I think that for the case of the usual vision-agent subsampling, a small value here usually returns better results.

Reply:
It does but it was timing out on too many examples with the small chunk_length. If we have improved the latency of the endpoint I can change it back down.